src/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from "next/link";
+import { Router, useRouter } from "next/router";
+import React from "react";
+import { TeamCardI } from "./team-interfaces";
+
+const TeamCard: React.FC<TeamCardI> = ({
```

Comment:
this 
```tsx
const TeamCard: React.FC<TeamCardI> = ({
	image,
	name,
	position,
	description,
	socials,
	email,
	phone,
	address,
})
```
should be written as 

```tsx
const TeamCard: React.FC<{ team: TeamCardI }> = ({ team }) => {
	return (
//... jsx
)
}
```

I will tell you for the last time, destructuring props from the top level has some unintended effect if any of the values is not present on first load. I prefer you use some optional chaning syntax like `team?.image`, team?.name` etc to get the values incase the values are missing

Reply:
Understood sir.